templates/agent/package.json

```diff
@@ -38,6 +38,8 @@
 	"private": true,
 	"scripts": {
 		"build": "next build",
+		"db:dev:push": "prisma db push --schema ./prisma/dev.schema.prisma && prisma db seed --schema ./prisma/dev.schema.prisma",
```

Comment:
A few things from testing locally:
- Change `--schema ./path` to `--schema=./path`
- Change `dev.schema.prisma` to `dev/schema.prisma`
- Remove `db seed` command altogether (I couldn't get it to work with our TSConfig)